### PR TITLE
OkHttp 3 port

### PR DESCRIPTION
**Pull request by vzamanillo**
Port to OkHttp 3 based on @blazsolar Butter [PR](https://github.com/butterproject/butter-android/pull/110)

### CODE REVIEW

File: base/src/main/java/butter/droid/base/manager/youtube/YouTubeManager.java [hunk 1]
```diff
@@ -21,10 +21,10 @@
 
 package butter.droid.base.manager.youtube;
 
-import com.squareup.okhttp.Call;
-import com.squareup.okhttp.OkHttpClient;
-import com.squareup.okhttp.Request;
-import com.squareup.okhttp.Response;
+import okhttp3.Call;
+        import okhttp3.OkHttpClient;
```
Comment (blazsolar) [inline base/src/main/java/butter/droid/base/manager/youtube/YouTubeManager.java]: unnecessary tabs

File: base/src/main/java/butter/droid/base/providers/subs/open/OpenSubsProvider.java [hunk 2]
```diff
@@ -15,33 +15,38 @@
...
 import de.timroes.axmlrpc.XMLRPCClient;
 import de.timroes.axmlrpc.XMLRPCException;
 import de.timroes.axmlrpc.XMLRPCServerException;
+import okhttp3.OkHttpClient;
 
 public class OpenSubsProvider extends SubsProvider {
 
-    private String mApiUrl = "http://api.opensubtitles.org/xml-rpc";
-    private String mUserAgent = "Popcorn Time v1";//"Popcorn Time Android v1";
+    static final String API_URL = "http://api.opensubtitles.org/xml-rpc";
```
Comment (blazsolar) [inline base/src/main/java/butter/droid/base/providers/subs/open/OpenSubsProvider.java]: this should probably be static

Comment (vzamanillo) [inline base/src/main/java/butter/droid/base/providers/subs/open/OpenSubsProvider.java]: I think they are static

File: base/src/main/java/butter/droid/base/manager/youtube/YouTubeManager.java (hunk 1 above)
Comment (vzamanillo) [inline base/src/main/java/butter/droid/base/manager/youtube/YouTubeManager.java]: :+1:

File: base/src/main/java/butter/droid/base/providers/subs/open/OpenSubsProvider.java (hunk 2 above)
Comment (vzamanillo) [inline base/src/main/java/butter/droid/base/providers/subs/open/OpenSubsProvider.java]: Can not be private, both are needed for OpenSubsModule

**Pull request merged**